packages/assets-controllers/src/TokenDetectionController.test.ts

```diff
@@ -1329,7 +1937,9 @@ type WithControllerCallback<ReturnValue> = ({
 }) => Promise<ReturnValue> | ReturnValue;
 
 type WithControllerOptions = {
-  options?: Partial<ConstructorParameters<typeof TokenDetectionController>[0]>;
+  options?: Partial<
+    ConstructorParameters<typeof TokenDetectionController>[0]
+  > & { isKeyringUnlocked?: boolean };
```

Comment:
I'm curious if adding a new option here could confuse people reading the tests into thinking that the controller takes this option. Should this be a separate option?

Reply:
I wasn't sure where to put `isKeyringUnlocked`, but I like the idea of keeping `options` limited to controller constructor. 

Applied here: https://github.com/MetaMask/core/pull/3867/commits/95b34ac7ab409587a1b998ef41db5a9d57d6bdad